apps/build-tools/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "presentation-build-tools",
+  "version": "0.0.0",
+  "description": "Presentation build tools",
+  "private": true,
+  "bin": {
+    "extract-api": "bin/extract-api.cjs"
+  },
+  "scripts": {
+    "clean": "rimraf lib build"
+  },
+  "dependencies": {
+    "@itwin/build-tools": "^4.7.1",
```

Comment:
we aren't using `@itwin/build-tools`, are we?

Reply:
I am now only using it for the `simpleSpawn` utility (`extract-api.cjs`). Maybe it's not worth using that

Reply:
Missed that. I don't mind having the dependency, as long as it's used - let's keep it.